### PR TITLE
assorted nexusphp: update and tidy

### PR DESCRIPTION
**Pull request by ilike2burnthing**
Used `remove: a, b, font, img, span` as general clean up for `description` (apart from pttime, because of course there's always one). A few required an additional replace filter. Shouldn't cause any issues when applied to indexers I don't have accounts for, but might as well check.

Removed unused English login errors (Jackett uses default, which is never English).

Updated hddolby.

Added missing sort settings.

Other tidying.

### CODE REVIEW

File: src/Jackett.Common/Definitions/52pt.yml [hunk 1]
```diff
@@ -40,10 +40,6 @@ settings:
     type: checkbox
     label: Search freeleech only
     default: false
-  - name: info_tpp
```
Comment (garfield69) [inline src/Jackett.Common/Definitions/52pt.yml:43]: why did you remove the ttp? all the nexus have this field in their control panel.

Comment (garfield69) [reply src/Jackett.Common/Definitions/52pt.yml:43]: oh! never mind. became obvious when I looked at the complete view rather than just the updates

File: src/Jackett.Common/Definitions/audiences.yml [hunk 2]
```diff
@@ -163,5 +163,5 @@ search:
       text: 172800
     description:
       selector: td:nth-child(2)
-      remove: a, img
+      remove: a, b, font, img, span
```
Comment (garfield69) [inline src/Jackett.Common/Definitions/audiences.yml:166]: this reduces 
image
to just
image

Comment (garfield69) [reply src/Jackett.Common/Definitions/audiences.yml:166]: wtf where are my pictures?

File: src/Jackett.Common/Definitions/hddolby.yml [hunk 3]
```diff
@@ -166,5 +189,8 @@ search:
       text: 1.2
     description:
       selector: td.rowfollow:nth-child(2)
-      remove: a, img
+      remove: a, b, font, img, span
+      filters:
+        - name: replace
+          args: [" 剩余时间：", ""]
 # NexusPHP Standard v1.5 Beta 4
```
Comment (garfield69) [inline src/Jackett.Common/Definitions/hddolby.yml:196]: you may want to check that if the site allows profile language to be set to english then this may be 'will end in'

File: src/Jackett.Common/Definitions/nicept.yml [hunk 4]
```diff
@@ -175,5 +174,8 @@ search:
       text: 259200
     description:
       selector: td.rowfollow:nth-child(2)
-      remove: a, img
+      remove: a, b, font, img, span
+      filters:
+        - name: replace
+          args: ["剩余时间：", ""]
 # NexusPHP v1.7.29 2022-10-13
```
Comment (garfield69) [inline src/Jackett.Common/Definitions/nicept.yml:181]: check if profile allows english

Comment (ilike2burnthing) [reply src/Jackett.Common/Definitions/nicept.yml:181]: hddolby, nicept, and ourbits all have a language profile setting, but it's per session rather than permanent, so Jackett will always get the default.

**Review by garfield69 — CHANGES_REQUESTED** [review]
I think overall you may be stripping out too much from the description
I have found several sites where part of the description is the time that the freeleech is due to expire, and that time is missing with your current strategy.
I think you might be better to restore the description to just removing the a and img tags for all sites, as I don't know the impact on sites I don't have access to.

File: src/Jackett.Common/Definitions/nicept.yml (hunk 4 above)
Comment (garfield69) [reply src/Jackett.Common/Definitions/nicept.yml:181]: ok, but if you are having to strip the chinese 'will end in' from the description then you have removed the freeleech will expire timestamp too, which was a useful piece of the description that was available before.

**Comment by ilike2burnthing** [comment]
Ah, I was purposefully stripping out the freeleech end time as it wasn't part of the release description, but rather information about the torrent.

In terms of other torrent information I was trying to strip out to make it clearer:
- `(NEW)` and `[Hot]` text - `b` and `font`
- IMDB and Douban ratings or `N/A` - `span`
- the `INT`, `HDR`, `REQ`, and various language coloured tags - `span`

The `will end in ...` can be `b`, `font`, or `span`, or some combination.

Put that all back in? Easily enough done.

**Comment by garfield69** [comment]
Yeah, Nah. I've already fixed the few where the whole chinese title got removed leaving just the stub 'will end in', so for the rest, if you are happy with what you've ended up with, then go ahead and publish. If anyone complains we can individually amend the indexer later.
